src/test/java/lottoTest/LottoStoreTest.java

```diff
@@ -0,0 +1,28 @@
+package lottoTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import lotto.LottoStore;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class LottoStoreTest {
+
+    @Test
+    @DisplayName("구매 비용은 1000의 배수여야 합니다. 아닐 시 IllegalArgumentException")
+    void INPUT_MULTIPLIES_OF_1000() {
+        assertThat(LottoStore.sellLotto(14000)).isEqualTo(14);
+
+        assertThatThrownBy(() -> LottoStore.sellLotto(13500))
+            .isInstanceOf(IllegalArgumentException.class);
```

Comment:
```suggestion
        assertThat(LottoStore.sellLotto(14000)).isEqualTo(14);

        assertThatThrownBy(() -> LottoStore.sellLotto(13500))
            .isInstanceOf(IllegalArgumentException.class);
```

[테스트당 개념 하나](https://github.com/Yooii-Studios/Clean-Code/blob/master/Chapter%2009%20-%20%EB%8B%A8%EC%9C%84%20%ED%85%8C%EC%8A%A4%ED%8A%B8.md#%ED%85%8C%EC%8A%A4%ED%8A%B8%EB%8B%B9-%EA%B0%9C%EB%85%90-%ED%95%98%EB%82%98)
두가지의 개념이 들어가진것 같아요~